demos/src/main.mustache

```diff
@@ -1,11 +1,29 @@
-<div class="demo-container--{{card-image-alignment}}">
-	<div class="o-card" data-o-component="o-card">
-		<div class="o-card__container">
-			<h2>Demo card</h2>
-			<p>My demo card</p>
-		</div>
-		{{#card-image-alignment}}
-			<img src="https://image.webservices.ft.com/v1/images/raw/http%3A%2F%2Fim.ft-static.com%2Fcontent%2Fimages%2Fa60ae24b-b87f-439c-bf1b-6e54946b4cf2.img?source=o-card-demo" class="o-card__image--{{card-image-alignment}}" alt="demo image"/>
-		{{/card-image-alignment}}
+<div class="demo-container demo-container--{{card-modifier}}">
+	<div class="o-card o-card--{{card-modifier}} o-card--stand-alone" data-o-component="o-card">
```

Comment:
Might be worth adding the `o-registry-demo-chunk` class to this <div> so that in the html the registry shows, the outer div doesn't appear